app/controllers/issues_controller.rb

```diff
@@ -1,5 +1,6 @@
 class IssuesController < AuthenticatedController
   include ActivityTracking
+  include CachedColumns
```

Comment:
should the concern be smarter? and hook on after_actions perhaps for #update or #create?

Reply:
There's risk involved with that. Comment coming in Pacifica.